bus/client/alerts.go

```diff
@@ -10,15 +10,13 @@ import (
 )
 
 // Alerts fetches the active alerts from the bus.
-func (c *Client) Alerts(opts alerts.AlertsOpts) (alerts []alerts.Alert, err error) {
+func (c *Client) Alerts(opts alerts.AlertsOpts) (resp alerts.AlertsResponse, err error) {
 	values := url.Values{}
-	if opts.Offset != 0 {
-		values.Set("offset", fmt.Sprint(opts.Offset))
-	}
+	values.Set("offset", fmt.Sprint(opts.Offset))
```

Comment:
why remove the surrounding `if opts.Offset != 0 {`

Reply:
To make sure we always trigger the new endpoint to get the new response type. Referencing the same route twice makes jape very unhappy so I just stuck to the new one.